CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 ### Added
 - De novo assembly alignment file load and display
 - Paraphase bam-let alignment file load and display
+- Parsing and showing ClinVar oncogenity criteria: `ONC`, `ONCREVSTAT` and `ONCDN`
```

Comment:
Leaving out ONCCONF for now, because the example didn't contain the key (the key is still very rare on ClinVar) so the parsing can't be really tested with a real case. I suggest we leave it for later

Reply:
A-OK, that is a slightly bigger issue that needs looking into for the RD side as well.